### ISSUE TITLE
Failed to handle over 2000 lines log file.

### ISSUE THREAD

**Issue by hirakiuc** [embulk/embulk-output-elasticsearch#9]
Thanks, great plugin !

But I have a problem to load my log data to elasticsearch with this plugin.

When I use this output plugin with my log file which has about 190,000 lines, embulk always fail with `java.lang.ArrayIndexOutOfBoundsException`.

On the other hand, when I use `embulk-output-redshift` plugin with same log file, I can load to Redshift successfully.

How to fix my problem ?

**Comment by hirakiuc** [comment]
```
$ embulk --version
embulk 0.6.5
```

Stacktrace: https://gist.github.com/hirakiuc/efefcb523ee8ff39a4a1.

out config on config.yml 

```
out:
  type: elasticsearch
  nodes:
    - { host: xx.xx.xx.xx, port: xxxx }
  cluster_name: elasticsearch
  index: my_logs
  index_type: logs
```

**Comment by muga** [comment]
@hirakiuc sounds like a bug.. I will take a look later:-)

**Comment by hirakiuc** [comment]
This issue is resolved by pull request #10 .
